es-ds-docs/pages/molecules/progress.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+    <div>
+        <h1>
+            Progress
+        </h1>
+        <p class="pb-200">
+            Extended from <nuxt-link
+                href="https://v3.primevue.org/progressbar/"
+                target="_blank">
+                primevue progress
+            </nuxt-link>
+        </p>
+        <h2>
+            Three Steps
+        </h2>
+        <div class="mb-100">
+            <es-progress :value="0" />
+        </div>
+        <div class="mb-100">
+            <es-progress :value="33" />
+        </div>
+        <div class="mb-100">
+            <es-progress :value="67" />
+        </div>
+        <div class="mb-100">
+            <es-progress :value="100" />
+        </div>
+        <h2>
+            Animated Transition
+        </h2>
+        <div class="mb-100">
+            <es-progress :value="value" />
+        </div>
+        <div class="mb-100">
+            <es-button
```

Comment:
As with the progress circle, we shouldn't have any references to es-button at the moment, since it causes hydration errors

Reply:
Got it, removed `es-button` from `progress.vue`